### PR TITLE
PS-1543 LP #1376987: [DOC] Add details on the conjunction of innodb_corrupt_table_action and innodb_force_recovery (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged with 5.7 version

### CODE REVIEW

File: doc/source/reliability/innodb_corrupt_table_action.rst [hunk 1]
```diff
@@ -23,8 +23,7 @@ running in the background.
 
 If the :variable:`innodb_force_recovery` variable is set to a low value
 and you expect the server to crash,
```
Comment (satya-bodapati) [inline doc/source/reliability/innodb_corrupt_table_action.rst:25]: with innodb_force_recovery set to low value, the intention is to not crash. Infact provide levels of recoverability after a crash and startup failures.

Please check this once: https://dev.mysql.com/doc/refman/8.0/en/forcing-innodb-recovery.html

File: doc/source/reliability/innodb_corrupt_table_action.rst [hunk 2]
```diff
@@ -23,8 +23,7 @@ running in the background.
...
-it may still be running due to
-a non-default value of the :variable:`innodb_corrupt_table_action` variable.
+the server may continue to run due to a non-default value of the :variable:`innodb_corrupt_table_action` variable.
```
Comment (satya-bodapati) [inline doc/source/reliability/innodb_corrupt_table_action.rst]: with innodb_force_recovery set to low value, the intention is to not crash. Infact it provides levels of recoverability after a crash and startup failures.

Please check this once: https://dev.mysql.com/doc/refman/8.0/en/forcing-innodb-recovery.html

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: doc/source/reliability/innodb_corrupt_table_action.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/reliability/innodb_corrupt_table_action.rst]: rewritten

File: doc/source/reliability/innodb_corrupt_table_action.rst [hunk 3]
```diff
@@ -21,10 +21,7 @@ which considerably reduces
 the effect of |InnoDB| subsystems
 running in the background.
 
-If the :variable:`innodb_force_recovery` variable is set to a low value
-and you expect the server to crash,
-it may still be running due to
-a non-default value of the :variable:`innodb_corrupt_table_action` variable.
+If the :variable:`innodb_force_recovery` option is <4, corrupted pages are lost and the server may continue to run due to the :variable:`innodb_corrupt_table_action` variable having a non-default value.
```
Comment (satya-bodapati) [inline doc/source/reliability/innodb_corrupt_table_action.rst:24]:
```suggestion
If the :variable:`innodb_force_recovery` option is < 4, corrupted pages are lost and the server may continue to run due to the :variable:`innodb_corrupt_table_action` variable having a non-default value.
```

**Review by satya-bodapati — APPROVED** [review]
Looks good. Please use the squash and merge option

**Comment by satya-bodapati** [comment]
> Looks good. Please use the squash and merge option

is this a merge from 5.7? then you shouldn't use squash and merge.
If not, you can use it

**Pull request merged**